app/models/org_identifier.rb

```diff
@@ -17,14 +17,33 @@
 #
 
 class OrgIdentifier < ActiveRecord::Base
+  include ValidationMessages
+
+  # ================
+  # = Associations =
+  # ================
+
   belongs_to :org
   belongs_to :identifier_scheme
-  
+
+  # ===============
+  # = Validations =
+  # ===============
+
   # Should only be able to have one identifier per scheme!
-  validates_uniqueness_of :identifier_scheme, scope: :org
-  
-  validates :identifier, :org, :identifier_scheme, presence: {message: _("can't be blank")}
-  
+  validates :identifier_scheme_id, uniqueness: { scope: :org_id,
+                                                 message: UNIQUENESS_MESSAGE }
+
+  validates :identifier, presence: { message: PRESENCE_MESSAGE }
```

Comment:
This one should be unique for the scheme. (e.g. one org's shibboleth id cannot be the same as any others)